release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* [PSAAS-20398] update dependencies in requirements
```

Comment:
I believe it would be good to mention that requirements were updated to resolve security vulnerabilities. Otherwise, the customers won't really know why should they update the connector. Maybe sth like:
```suggestion
* Updated dependencies to resolve security vulnerabilities [PSAAS-20398]
```